src/com/ableton/VirtualEnv.groovy

```diff
@@ -126,6 +132,15 @@ class VirtualEnv implements Serializable {
     script.dir(venvRootDir) { script.deleteDir() }
   }
 
+  /**
+   * Run a closure body inside of the virtualenv.
+   *
+   * @param body Closure body to execute.
+   */
+  void inside(Closure body) {
+    script.withEnv(["PATH+EXTRA=${venvBinDir}"]) { body() }
```

Comment:
We can write whatever we want after `PATH+` so let's use something that makes a bit more sense (like `VIRTUALENV` or `VENVBIN`).

Reply:
Where is the documentation on `PATH+WHATEVER`?

Reply:
https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#withenv-set-environment-variables

Reply:
Fixed in a153738.